pkg/agent/params.go

```diff
@@ -219,8 +219,8 @@ func assignKubernetesParameters(properties *datamodel.Properties, parametersMap
 			addValue(parametersMap, "containerRuntime", kubernetesConfig.ContainerRuntime)
 			addValue(parametersMap, "containerdDownloadURLBase", cloudSpecConfig.KubernetesSpecConfig.ContainerdDownloadURLBase)
 			addValue(parametersMap, "cniPluginsURL", cloudSpecConfig.KubernetesSpecConfig.CNIPluginsDownloadURL)
-			addValue(parametersMap, "vnetCniLinuxPluginsURL", kubernetesConfig.GetAzureCNIURLLinux(cloudSpecConfig))
-			addValue(parametersMap, "vnetCniWindowsPluginsURL", kubernetesConfig.GetAzureCNIURLWindows(cloudSpecConfig))
+			addValue(parametersMap, "vnetCniLinuxPluginsURL", kubernetesConfig.GetAzureCNIURLLinux(*cloudSpecConfig))
```

Comment:
looks like this should also be updated GetAzureCNIURLLinux

Reply:
To keep it consistent using pointers

Reply:
Yeah, sure thing. Updated to use pointers everywhere.